.circleci/config.yml

```diff
@@ -17,34 +17,51 @@ jobs:
     docker:
       - image: cimg/openjdk:8.0
     <<: *lint_steps
+
+  # JDK 8 (LTS) - 31 Dec 2030 (extended support)
```

Comment:
If there's a good reference you have for these LTS dates, may be useful to drop in here

Reply:
I used https://endoflife.date/oracle-jdk to work them out.